autorest/codegen/models/enum_schema.py

```diff
@@ -76,13 +74,14 @@ def get_python_type_annotation(self) -> str:
         """
         return f'Union[str, \"{self.enum_type}\"]'
 
-    def get_python_type(self, namespace: str) -> str:
+    def get_python_type(self, namespace: str) -> str:  # pylint: disable=signature-differs
```

Comment:
That's unexpected, why would they differ?

Reply:
got rid of all signature-differs suppression but adding namespace as a required parameter for BaseSchema's from_yaml